src/components/Steps/Step.js

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react'
+import cx from 'classnames'
+
+import checkmark from './../../assets/checkmark.svg'
+
+import styles from './Steps.module.scss'
+
+const Step = ({
+  className,
+  active,
+  status = 'wait',
+  stepNumber,
+  disabled,
+  description,
+  title,
+  icons,
+  stepIndex,
+  onStepClick,
+  onClick,
+  ...restProps
+}) => {
+  const handleClick = (...args) => {
+    if (onClick) {
+      onClick(...args)
+    }
+
+    onStepClick(stepIndex)
+  }
+
+  const renderIconNode = () => {
+    if (icons && icons.process && status === 'process') {
```

Comment:
Guys it's a question, probably due to my small practice it's not a right way but :)
Could we use this way now ```icons?.process?.status === 'process'```
cc @DmitriVanGuard @Woafflation @peratik ?

Reply:
`status ` is not a property of `icons`

Reply:
ah yep.